src/net/sourceforge/kolmafia/swingui/panel/CommandDisplayPanel.java

```diff
@@ -24,9 +28,14 @@ public class CommandDisplayPanel extends JPanel implements FocusListener {
   private final AutoHighlightTextField entryField;
   private final JButton entryButton;
 
+  private final String preference;
+  private final String DELIMITER = "#";
```

Comment:
while # is unlikely to appear, why not unit separator (␟) or something even less likely to appear?

Reply:
Forgot/didn't know it existed.  But I went for the # because it was also already on most keyboards and easily managed in dumb, ASCII text editors.  We can discuss but human readability and editing or preference files are of concern to me.